### PR TITLE
feat: adding vk social icon

### PR DESCRIPTION
**Pull request by ogandose**
Adding icon for vk.

### CODE REVIEW

**Review by lpoulter — APPROVED** [review]

**Comment by lpoulter** [comment]
Feel free to merge and release. You should be able to trigger this action to trigger a new version of Axiom
https://github.com/BrandwatchLtd/axiom-react/actions?query=workflow%3A%22Publish+to+Npm%22

File: packages/axiom-materials/src/icon_svgs/vk-mono.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+  <g fill="none" fill-rule="evenodd">
+    <path fill="#6F6F6F" fill-rule="nonzero" d="M-1.18423789e-15,7.66666667 C-1.18423789e-15,4.05256667 -1.18423789e-15,2.24551333 1.12275667,1.12275667 C2.24551333,0 4.05256667,0 7.66666667,0 L8.33333333,0 C11.9474333,0 13.7545,0 14.8772333,1.12275667 C16,2.24551333 16,4.05256667 16,7.66666667 L16,8.33333333 C16,11.9474333 16,13.7545 14.8772333,14.8772333 C13.7545,16 11.9474333,16 8.33333333,16 L7.66666667,16 C4.05256667,16 2.24551333,16 1.12275667,14.8772333 C-1.18423789e-15,13.7545 -1.18423789e-15,11.9474333 -1.18423789e-15,8.33333333 L-1.18423789e-15,7.66666667 Z"/>
```
Comment (Flawwles) [inline packages/axiom-materials/src/icon_svgs/vk-mono.svg:3]: The fill should be #000000
Not a big deal but if this icon is used in a button or needs to inherit the theme colour it won't work

Comment (ogandose) [reply packages/axiom-materials/src/icon_svgs/vk-mono.svg:3]: I'll update it in a new pr 👍

Comment (ogandose) [reply packages/axiom-materials/src/icon_svgs/vk-mono.svg:3]: https://github.com/BrandwatchLtd/axiom-react/pull/992

**Pull request merged**